Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -228,7 +251,9 @@ protected class ClassWriter : IClassWriter {
 
     protected override void DeclareLocalVar(string name, Type type, IToken tok, bool leaveRoomForRhs, string rhs,
       ConcreteSyntaxTree wr) {
-      throw new NotImplementedException();
+      wr.Write("{0}{1}", name, type != null ? "=" + TypeName(type, wr, tok) + "()" : "");
```

Comment:
The else part can be used to define object of a class.

Reply:
Can you write `$"={TypeName(type,wr,tok)}()"`? It's more readable (less double quotes and plus symbols.

Reply:
edited.